### ISSUE TITLE
Which version of Manim does it work with?

### ISSUE THREAD

**Issue by tarkesdora** [CaftBotti/manim_pi_creatures#1]
Does this work only in Old Manim cairo version ? How about in Manim CE version?

**Comment by CaftBotti** [comment]
Currently, it works only on cairo-backend, because cairo-backend is the only version with the function of creating pi creature scenes. Pi creatures have been removed in recent versions of manim, both 3b1b/manim and community version.

But there's a good solution: rewrite the four files in the for_3b1b_videos folder as the code that is runnable for ManimCE. And it will certainly produce the same effect as in cairo-backend.

**Comment by tarkesdora** [comment]
Thanks for responding. Similarly, if I create SVG files for other greek letters, it would function the same with this code?

**Comment by CaftBotti** [comment]
Yes, if you replace the pi's body with SVG paths of greek letters using editors like Adobe Illustrator, you will get a Tau creature scene, etc. In fact, you can change your pi into any shapes you like. 

But there is one thing you have to notice: you should only "change" the paths of body, eyes and mouth, but don't "add" or "delete" paths. For example, please don't draw a new path on the SVG file even a very small draw. This is because the change of the amount of paths has a possibility of making your new creature look "weird".

**Comment by groblerfg** [comment]
Is there maybe a how to guide, to explain how to implement this on manimCE. In your README, you refer to replace assets/piCreatures. From manimCE I don't see a assets folder.
